### DISCUSSION TITLE
Cell Remove

### DISCUSSION THREAD

**Discussion by nahidrahmati** [UvaCsl/HemoCell#97] [General discussion]
Hi, 
I have changed the domain geometry by defining a cosine curved pipe with STL. I have a spherical cell withing the domain close to the cell wall. Although I have activate the repulsion force and its magnitude is large enough, but the cell is being removed. The warning that is shown on log file, give me the location of particle that is not in local domain, while when I check that specific location, it is within the fluid domain. Is there any difference between locale domain and fluid lattices? If yes, how can I define the local domain in a way to be exactly similar to the fluid domain?

**Comment by gzavo** [comment]
Hi,

If you search the 'Discussions' for the word "remove" you will find several conversations on why can cells be removed and what can you try to fix it.

Bests,
Gabor
